admin/admin.php

```diff
@@ -194,22 +194,24 @@ function tlms_showWarnings(){
 		echo '<div class="error notice">'.$message.'</div>';
 	}
 }
-
-if((!get_option('tlms-domain') && !get_option('tlms-apikey')) && (empty($_POST['tlms-domain']) && empty($_POST['tlms-apikey']))){
-	tlms_logError('<p><strong>'.__('You need to specify a TalentLMS domain and a TalentLMS API key.', 'talentlms').'</strong>'
-		.sprintf(__('You must <a href="%1$s">enter your domain and API key</a> for it to work.', 'talentlms'), 'admin.php?page=talentlms-setup').'</p>');
-}
-else{
-	try{
-		TalentLMS::setDomain(get_option('tlms-domain'));
-		TalentLMS::setApiKey(get_option('tlms-apikey'));
-
-		tlms_getCourses();
-		tlms_getCategories();
+tlms_getCourses();
```

Comment:
Did you left this here on purpose? I think `tlms_getCourses` is also called always now

Reply:
It was a mistake. Fixed in https://github.com/epignosis/talentlms-wordpress/pull/16/commits/a1a1ae9c28d0b05da0690d10ca351f44dc581d5f